z = [4, 1, 3]

# Will be printing Index Error exception
print(z[4])


z = int("jim")

# Prints ValueError issue
print(z)


codes = dict(gb=44, us=1, no=47, fr=33, es=34)
# Print the KeyError
print(codes['de'])
